data/json/npcs/talk_tags.json

```
@@ -1032,7 +1230,8 @@
     "text": [
       "I can't outrun it!  I'm going to kill it!",
       "<swear>!  Die, you <name_b>!  I want to live!",
-      "My feet failed me!  Arms don't fail me!"
+      "My feet failed me!  Arms don't fail me!",
```

```suggestion
      "My feet failed me!  Arms, don't fail me!",
```
Again, not your change, but grammatical error.